website/docs/d/is_virtual_endpoint_gateway_ips.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_is_virtual_endpoint_gateway_ips"
+sidebar_current: "docs-ibm-datasource-is-virtual-endpoint-gateway-ips"
+description: |-
+  Manages IBM Cloud Infrastructure virtual endpoint gateway ips.
+---
+
+# ibm_is_virtual_endpoint_gateway_ips
+
+Import the details of an existing IBM Cloud Infrastructure virtual endpoint gateway ips as a read-only data source. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_is_virtual_endpoint_gateway_ips" "data_test1" {
+  gateway_id     = ibm_is_virtual_endpoint_gateway.endpoint_gateway.id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- gateway_id(Required,string)- Endpoint gateway ID
```

Comment:
check other docs and change this line accordingly.

Reply:
handled